src/main/java/org/skife/jdbi/v2/BeanMapper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2004 - 2014 Brian McCallister
+ * Copyright (C) 2004 - 2015 Brian McCallister and others
```

Comment:
I'd prefer we just got rid of the license text, personally.

On Thu, Sep 17, 2015 at 3:59 PM, Matthew Hall <notifications@github.com>
wrote:

> In src/main/java/org/skife/jdbi/v2/BeanMapper.java
> <https://github.com/jdbi/jdbi/pull/164#discussion_r39755578>:
>
> > @@ -1,5 +1,5 @@
> >  /*
> > - * Copyright (C) 2004 - 2014 Brian McCallister
> > + * Copyright (C) 2004 - 2015 Brian McCallister and others
>
> I take that to mean that every file touched in this PR should have updated
> license text?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdbi/jdbi/pull/164/files#r39755578>.
>